solr/contrib/ltr/example/train_and_upload_demo_model.py

```diff
@@ -10,8 +10,8 @@
 
 solrQueryUrl = ""
 
-def generateQueries(config):
-        with open(config["userQueriesFile"]) as input:
+def generateQueries(userQueriesFile, config):
+        with open(userQueriesFile) as input:
```

Comment:
To make it clearer that the config is used only for the generation work itself. Actually, let me go and try if some sort of functor approach is possible i.e. something like
```
  def generateQueries(userQueriesFile, urlGenerator):
  ...
  solrQuery = urlGenerator(searchText,docId,score,source)
```
That way then generateQueries doesn't get to see config at all.